hw3/03-react-app/src/search.js

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+import axios from 'axios';
+
+export default function Search() {
+  const [name, setName] = useState('');
+  const [foundName, setFoundName] = useState('');
+  const [foundImgage, setFoundImage] = useState('');
+
+  const url = 'https://thronesapi.com/api/v2/Characters';
+
+  async function searchName(name) {
+    try {
+      const response = await axios.get(url);
```

Comment:
I would suggest the same here, fetch the data once in App.js and use it in the other files to avoid redundancy.

Reply:
Updated in PR